### PR TITLE
Update docs to stop referencing MyGet.

### PR DESCRIPTION
**Pull request by simeoncran**
We're now using Azure Devops instead of MyGet for CI builds.

### CODE REVIEW

File: Lottie-Windows/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The Lottie-Windows project generates a NuGet package for use by Windows apps.
 
 ## Package locations
 * The [latest release and pre-release versions are on NuGet](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI.Lottie).
-* The [latest CI builds are on MyGet](https://dotnet.myget.org/feed/uwpcommunitytoolkit/package/nuget/Microsoft.Toolkit.Uwp.UI.Lottie).
+* The NuGets for the latest CI builds are published to Azure DevOps. Links here for the [main branch](https://pkgs.dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging/WindowsCommunityToolkit-MainLatest/nuget/v3/index.json) and for [PRs](https://pkgs.dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging/WindowsCommunityToolkit-PullRequests/nuget/v3/index.json). See the [Windows Community Toolkit wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki/Preview-Packages) for details.
```
Comment (michael-hawker) [inline Lottie-Windows/README.md:8]:
```suggestion
* The NuGets for the latest CI builds are published to Azure DevOps. Links here for the [main branch](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging?_a=feed&feed=WindowsCommunityToolkit-MainLatest) and for [PRs](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging?_a=feed&feed=WindowsCommunityToolkit-PullRequests). See the [Windows Community Toolkit wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki/Preview-Packages) for details.
```

These are the landing pages. You can also see the change we made to our badges in our readme if you want to follow that format instead.

Just FYI, we don't have the PRs for your repo setup yet, but I can enable them if you'd like.

Comment (simeoncran) [reply Lottie-Windows/README.md:8]: Please do enable PRs. They're handy.

---
In reply to: [519997728](https://github.com/windows-toolkit/Lottie-Windows/pull/381#discussion_r519997728) [](ancestors = 519997728)

Comment (simeoncran) [reply Lottie-Windows/README.md]: @michael-hawker  - please do. BTW, I couldn't @ you in CodeFlow... weird.

File: Lottie-Windows/README.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ The Lottie-Windows project generates a NuGet package for use by Windows apps.
 
 ## Package locations
 * The [latest release and pre-release versions are on NuGet](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI.Lottie).
-* The [latest CI builds are on MyGet](https://dotnet.myget.org/feed/uwpcommunitytoolkit/package/nuget/Microsoft.Toolkit.Uwp.UI.Lottie).
+* The NuGets for the latest CI builds are published to Azure DevOps. Links here for the [main branch](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging?_a=package&feed=WindowsCommunityToolkit-MainLatest&protocolType=NuGet&package=Microsoft.Toolkit.Uwp.UI.Lottie) and for [PRs](https://dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging?_a=package&feed=WindowsCommunityToolkit-PullRequests&protocolType=NuGet&package=Microsoft.Toolkit.Uwp.UI.Lottie).
```
Comment (derobinsMSFT) [inline Lottie-Windows/README.md:8]: >  [](start = 468, length = 1)

Nit: extra space.

File: LottieGen/README.md [hunk 3]
```diff
@@ -4,10 +4,23 @@ LottieGen is a tool for generating C#, C++, and other outputs from Lottie / Body
 
 LottieGen requires [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) or later.
 
-# Installing
+# Run without installation
+
+Local builds can be run directly. Just install [.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1), build the LottieGen.sln solution and run the output.
+
+    f:\GitHub\Lottie-Windows\LottieGen\bin\AnyCpu\Debug\netcoreapp3.1\lottiegen.exe
```
Comment (derobinsMSFT) [inline LottieGen/README.md:11]: >AnyCpu [](start = 43, length = 6)

Nit: place in angle brackets (<>) to make clear that this is not an actual directory, and needs to be replace.

Comment (simeoncran) [reply LottieGen/README.md:11]: That's actually the directory. AnyCpu is the name of the architecture target for .NET executables that contain no architecture-specific code (i.e. they are MSIL only).

---
In reply to: [520835375](https://github.com/windows-toolkit/Lottie-Windows/pull/381#discussion_r520835375) [](ancestors = 520835375)

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

**Pull request merged**